code/hud/hudparse.cpp

```diff
@@ -4276,6 +4282,13 @@ void load_gauge_wingman_status(gauge_settings* settings)
 			grow_mode = 2;
 	}
 
+	if (optional_string("Wingname Align Mode:")) {
+		if (optional_string("Left"))
+			wingname_align_mode = 1;
+		else if (optional_string("Right"))
+			wingname_align_mode = 2;
```

Comment:
It would be useful to add an `else` case with a warning (perhaps if the string wasn't "Center")